src/auth/reference_modules/oidc.py

```diff
@@ -57,12 +56,18 @@ def _parse_response(response):
 def _load_role_mappings(raw_role_mappings: str) -> dict:
     if raw_role_mappings:
         role_mapping = {}
-        raw_role_mappings = raw_role_mappings.split(";")
+        raw_role_mappings = raw_role_mappings.strip().split(";")
         for mapping in raw_role_mappings:
-            idp_role, mg_role = mapping.split(":")
+            mapping_list = mapping.split(":")
+            if len(mapping_list) == 0:
+                continue
+            if len(mapping_list) != 2:
+                raise ValueError(f"Invalid role mapping: {mapping}")
+            idp_role, mg_role = mapping_list
             role_mapping[idp_role.strip()] = mg_role.strip()
         return role_mapping
-    return {}
+
+    raise ValueError("Missing role mappings")
```

Comment:
Is this an error?
Do we have some default behavior?

Reply:
You cannot use SSO without role mappings.
This would otherwise fail later and throw `Cannot map role {role} to Memgraph role`.
So this is unecessary in a way but I would say missing the whole env variable deserves a separate error.